### PR TITLE
Dev ai8x examples

### PR DESCRIPTION
**Pull request by ozersa**
MAX78000/02 examples reviewed.
Detail [here](https://confluence.maxim-ic.com/display/MSW/MSDK+Example+Validation+and+Adding+Missing+Boards)
Unresolved jira tickets 
[MAX78000](https://jira.maxim-ic.com/browse/MSDK-660)
[MAX78002](https://jira.maxim-ic.com/browse/MSDK-661)

### CODE REVIEW

**Review by sihyung-maxim — APPROVED** [review]

**Review by Jake-Carter — COMMENTED** [review]
Looks good, nice catch on the missing ram linkerfile.  Not sure where or when those project.mk files got reset... 

One minor comment to resolve for AI85 Flash main.c

File: Examples/MAX78000/Flash/main.c [hunk 1]
```diff
@@ -249,13 +248,7 @@ int main(void)
...
     printf("writing and verifying a test pattern from\n");
     printf("ADDR: 0x%x to ADDR: 0x%x\n", MXC_FLASH_MEM_BASE, MXC_FLASH_MEM_BASE + TESTSIZE);
-#ifdef BOARD_FTHR_REVA
-    printf("Push SW1 to begin\n");
-#else
-#ifdef BOARD_EVKIT_V1
-    printf("Push PB1 to begin\n");
-#endif
-#endif
+    printf("Press Push Button 1 (PB1/SW1) to begin\n");
```
[the remark this answers is not in the document]
Comment (ozersa) [reply Examples/MAX78000/Flash/main.c:251]: The purpose to simplify code, the more #if case, the more unreadable code,
I believe the updated printf messages cover two board case too.

**Review by sihyung-maxim — APPROVED** [review]

**Review by Jake-Carter — APPROVED** [review]
Looks good

**Comment by lorne-maxim** [comment]
@ozersa, this PR is merged.  You can delete the branch when you're ready.

**Pull request merged**